src/config.ts

```diff
@@ -27,7 +27,11 @@ let config: Config | undefined
 
 export function getConfig() {
   if (!config) {
-    if (!fs.existsSync(rcPath))
+    const { packageManager = '' } = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'package.json'), 'utf8') || '{}')
```

Comment:
We might want to use `find-up` to search the package.json, and check if it exist before reading.